unittests/test_schedulers.py

```diff
@@ -166,6 +166,11 @@ def test_poll_before_submit(self):
         self.testjob.prepare(self.commands, self.environs)
         self.assertRaises(JobNotStartedError, self.testjob.finished)
 
+    def test_no_empty_lines_in_preamble(self):
```

Comment:
This should be part of the `_TestJob` base class. Subclasses need not re-implement this, since it should be the same everywhere.